app/controllers/api/connect/base_controller.rb

```diff
@@ -29,7 +29,7 @@ def authenticate_system
       else
         logger.info "Could not find system with login '#{login}' and password '#{password}'"
         untranslated = N_('Invalid system credentials')
-        respond_with_error({ message: untranslated, localized_message: _(untranslated), status: :unauthorized })
+        raise ActionController::TranslatedError.new(error: untranslated, localized_error: _(untranslated), status: :unauthorized)
```

Comment:
Actually, since in `TranslatedError` we have  `def initialize(error: 'Error occurred', localized_error: _(error), #....`, when `localized_error` is just the same as `error` but with `_(...)`, we can just ommit the `localized_error` option:

```
raise ActionController::TranslatedError.new(error: untranslated, status: :unauthorized) 
```